common/djangoapps/course_modes/views.py

```diff
@@ -43,6 +44,24 @@
 
 LOG = logging.getLogger(__name__)
 
+# Namespace for course_modes waffle flags.
+WAFFLE_FLAG_NAMESPACE = LegacyWaffleFlagNamespace(name='course_modes')
+
+# .. toggle_name: course_modes.use_new_track_selection
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: This flag enables the use of the new track selection template for testing purposes before full rollout
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2021-8-23
+# .. toggle_target_removal_date: None
+# .. toggle_tickets: REV-2133
+# .. toggle_warnings: This temporary feature toggle does not have a target removal date.
+VALUE_PROP_TRACK_SELECTION_FLAG = LegacyWaffleFlag(
+    waffle_namespace=WAFFLE_FLAG_NAMESPACE,
+    flag_name='use_new_track_selection',
+    module_name=__name__,
+)
+
```

Comment:
Just to confirm my understanding: how does this one waffle flag relate to the first one that was already made? (I only see new lines, not changes to the other one). Also, the commented lines are just notes on what values to use when you add the flag?